### PR TITLE
Fix/general fix

### PR DESCRIPTION
**Pull request by AlviseSembenico**

### CODE REVIEW

**Comment by AlviseSembenico** [comment]
Jenkins running http://jenkins.maykin.nl/job/VNG%20Github/job/api-testvoorziening/view/change-requests/job/PR-161/

**Comment by AlviseSembenico** [comment]
I think it looks neat now
Screenshot from 2019-05-09 17-46-10

File: src/vng/js/components/statusbar/statusbar.js [hunk 1]
```diff
@@ -2,19 +2,23 @@ import $ from 'jquery';
...
     $.ajax({
         url: window.url + obj.attr(window.attr_name) + '/',
         success: (data, textStatus, jqXHR) => {
-            var session = data
-            if (session.deploy_status) {
-                obj.html(`${session.deploy_status}: ${session.deploy_percentage}%<br />
-                <div style='background-color: green; width: ${ session.deploy_percentage}%; height: 20px;'></div>`)
-                if (session.deploy_percentage < 100) setTimeout(() => {
+            var session = data;
+            if (session[window.percentage]) {
```
Comment (alextreme) [inline src/vng/js/components/statusbar/statusbar.js]: If you want to see a horrified Sven, ask him about setting variables on window.

But it works for me.

File: src/vng/servervalidation/templates/servervalidation/failed_test_email.html [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+Failed
+<ul>
+    {% for s in failure %}
+        <li>
+            https://{{domain}}{% url 'server_run:server-run_detail_uuid' s.0.uuid %}
+        </li>
+    {% endfor %}
+</ul>
+<br>
+Successfull:
```
Comment (alextreme) [inline src/vng/servervalidation/templates/servervalidation/failed_test_email.html]: https://www.merriam-webster.com/dictionary/successful

File: src/vng/servervalidation/forms.py [hunk 3]
```diff
@@ -46,8 +46,8 @@ def __init__(self, quantity=0, field_name='field', text_area=[], *args, **kwargs
         super().__init__(*args, **kwargs)
         for i in range(quantity):
             if isinstance(field_name, str):
-                self.fields['{}-{}'.format(field_name, i + 1)] = forms.URLField()
+                self.fields['{}-{}'.format(field_name, i + 1)] = forms.CharField(widget=forms.Textarea)
```
Comment (alextreme) [inline src/vng/servervalidation/forms.py]: Although I don't mind storing the URLs in a Textfield, doesn't setting the widget to a Textarea cause a multi-line input widget?

I'm fine with using the URLField() form field and storing the data in a Textfield

File: src/vng/js/components/statusbar/statusbar.js (hunk 1 above)
Comment (AlviseSembenico) [reply src/vng/js/components/statusbar/statusbar.js]: I am aware, (I dont open that code when he is looking) but i don't know a better way to communicate with a module (funtion) whose name has been hidden by gulp.

File: src/vng/servervalidation/forms.py (hunk 3 above)
Comment (AlviseSembenico) [reply src/vng/servervalidation/forms.py]: The problem is that now we have to handle not only url but general variables like the different jwt that i showed you. Therefore, a urlfield would complain about it

**Comment by alextreme** [comment]
Merge conflict and a failing test on jenkins

**Comment by alextreme** [comment]
Can be merged after fixing 'successfull' and having jenkins pass, nice work!

**Pull request merged**